Misc/NEWS.d/next/Core and Builtins/2022-10-28-14-52-55.gh-issue-98783.iG0kMs.rst

```diff
@@ -0,0 +1,2 @@
+Fix multiple crashes when ``str`` subclasses where used instead of ``str``
```

Comment:
Maybe mention that these were assertion failures in debug builds only?

Reply:
Done!